perception/sub8_perception/nodes/test_path_marker.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
Nothing should really need to import from this right? And yea, if anything breaks, add an __init__.py and update the setup.py.